index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>My portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="./assets/libraries/fontawesome/css/all.css" rel="stylesheet">
+        <link rel="stylesheet" href="./assets/css/style.css">
+    </head>
+    <body>
+        <div class="container">
+            <header class="header">
+                <div class="logo">
```

Comment:
- [ ] I would suggest using the `<a>` tag instead of a `<div>` tag. The common convention is to use the logo/main image as a link to the main page/index :wink:
Remember that is a good practice to use HTML5 semantic tags over `<div>`s according to the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) :memo: